src/components/Actions/Auth0Route.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Route, Redirect } from "react-router-dom";
+import { useAuth0 } from "@auth0/auth0-react";
+
+const Auth0ProtectedRoute = ({ Component, path, ...rest }: any) => {
+  const { isAuthenticated } = useAuth0();
```

Comment:
Don't we need to also do a check for `isLoading` here? Is it sufficient to check for just `isAuthenticated`?

Reply:
Just `isAuthenticated`.